target/linux/mpc85xx/files/arch/powerpc/boot/dts/hiveap-330.dts

```diff
@@ -53,17 +53,8 @@
 				};
 
 				partition@40000 {
-					reg = <0x40000 0x40000>;
-					label = "initrd";
-				};
-
-				partition@80000 {
-					reg = <0x80000 0x27c0000>;
-					label = "rootfs";
-				};
-
-				partition@2840000 {
-					reg = <0x2840000 0x800000>;
+					compatible = "openwrt,uimage", "denx,uimage";
```

Comment:
Doesn't the `compatible = "openwrt,uimage", "denx,uimage";` line need to be in the `firmware@0 {` partition so the DTB is correctly programmed in by sysupgrade?